### DISCUSSION TITLE
Decouple brightness from color temperature

### DISCUSSION THREAD

**Discussion by lorandignat** [basnijholt/adaptive-lighting#1066] [Ideas]
Hi! Would be nice to be able to decouple the brightness settings change from the color temperature change.

Example during winter the sun sets very early, and would be nice for the color temperature to follow the sun (e.g. get warmer as the sun sets), but I don't want to also start dimming it until much later. 

While this is somewhat possible with brightness_mode_time_dark, it would be nice to be able to change brightness_mode_time_dark based on the time of the year somehow, shorter during summer, longer during winter. Or just to be able to set a fixed time for when the dimming to start, and have the color temperature behave as is now.

**Comment by Benneboy** [comment]
YES!!! having the same wish at the mo. I have tried experimenting with two ALs attached to one light group, one with brightness disabled and the other with colour disabled. But this does not work. Only one takes control (or is successful in issuing its commands).

I would love to have colour (color) track sunrise-sunset and, personally choose the brightness times. However, for universal appreciation I think full separate timing functions for both is a must.
